README.md

```diff
@@ -79,70 +84,31 @@ In order to keep Alamofire focused specifically on core networking implementatio
 
 ### CocoaPods
 
-[CocoaPods](https://cocoapods.org) is a dependency manager for Cocoa projects. You can install it with the following command:
-
-```bash
-$ gem install cocoapods
-```
-
-> CocoaPods 1.1+ is required to build Alamofire 4.0+.
-
-To integrate Alamofire into your Xcode project using CocoaPods, specify it in your `Podfile`:
+[CocoaPods](https://cocoapods.org) is a dependency manager for Cocoa projects. For usage and installation instructions, visit their website. To integrate Alamofire into your Xcode project using CocoaPods, specify it in your `Podfile`:
 
 ```ruby
-source 'https://github.com/CocoaPods/Specs.git'
-platform :ios, '10.0'
-use_frameworks!
-
-target '<Your Target Name>' do
-    pod 'Alamofire', '~> 4.7'
-end
-```
-
-Then, run the following command:
-
-```bash
-$ pod install
+pod 'Alamofire', '~> 5.0.0.beta.1'
 ```
 
 ### Carthage
 
-[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks.
-
-You can install Carthage with [Homebrew](https://brew.sh/) using the following command:
-
-```bash
-$ brew update
-$ brew install carthage
-```
-
-To integrate Alamofire into your Xcode project using Carthage, specify it in your `Cartfile`:
+[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks. To integrate Alamofire into your Xcode project using Carthage, specify it in your `Cartfile`:
 
 ```ogdl
-github "Alamofire/Alamofire" ~> 4.7
+github "Alamofire/Alamofire" ~> 5.0.0.beta.1
 ```
 
-Run `carthage update` to build the framework and drag the built `Alamofire.framework` into your Xcode project.
```

Comment:
Are you assuming that people now know to do this? Or that they will get this from the Carthage documentation? It's perfectly fine by me to remove it, but I think it was originally added to help get people going and remind them not to forget that important step.

> Although I can completely understand if you feel it isn't the Alamofire README's responsibility to provide you with instructions on how to use Carthage.

Reply:
Actually, looks like the links are already there, if you're okay with this change in the PR.

Reply:
Makes perfect sense.